clients/geth/specular/Makefile

```diff
@@ -1,26 +1,64 @@
-.PHONY: geth bindings install
+.PHONY: install specular clean geth-docker contracts
 
+GOSRC = ./...
 GOBIN = ./build/bin
+# TODO use env variables
```

Comment:
maybe using env should be in this PR already, wdyt?

I just learned that we can use `include .env` in the Makefile to import the env variables ([docs](https://www.gnu.org/software/make/manual/html_node/Include.html))

Reply:
yeah but idt we can `include sbin/configure.sh`; we also can't run it from inside the Makefile since the variables won't be exported that way, so the only way is to probably convert `sbin/configure.sh` into a makefile

Reply:
my idea would have been to move all env vars into an `.env` file so we can just include that in the Makefile.

`sbin/configure.sh` can just run `export $(cat .env | xargs)` to preserve the current functionality

Reply:
please move them to an env file. other changes lgtm. also consider remove binding changes

Reply:
to be fixed later